test/unit/file_transfer_agent/file_util_test.js

```diff
@@ -50,3 +50,52 @@ describe('matching files by wildcard', function () {
   });
 
 });
+
+
+describe('varify only user read/write permission', function () {
+  let testFilePath;
+
+  before( async function () {
+    testFilePath = await testUtil.createTempFileAsync(os.tmpdir(), testUtil.createRandomFileName());
+  });
+
+  after(async function () {
+    await fsPromises.rm(testFilePath);
+  });
+
+  [
+    {
+      permission: '600',
```

Comment:
in code we change the suffix only - let's add the case with '600' prefixed with sth